examples/open-loop/plot_loop.py

```diff
@@ -1,19 +1,16 @@
-import sys
-
-sys.path.append('../')
-
```

Comment:
We can assume users have installed in a Python or Conda environment, so we can just import the `eyeloop` module :+1: